### ISSUE TITLE
Fix user menu links ads and my-ads

### ISSUE THREAD

**Issue by kevinmamaqi** [IT-Academy-BCN/ita-directory#828]

**Comment by rabbanirizwan** [comment]
can you provide details?

**Comment by kevinmamaqi** [comment]
Yes, there is no differenciation between the /ads and the /my-ads pages. One should bring all, and the other one just user ads.

Moreover, my-ads (user-ads) should have an edit button instead of an contact button.

**Comment by Bazza6** [comment]
After fixing the my-ads path, there are no ads displayed in 'Mis Anuncions'. Instead, an error appears.
To reproduce this issue you need first to modify the paths.js file.

Steps to reproduce:
1. Open paths.js and replace line 9 with: userAds: '/user-ads',
2. Save the change.
3. Go to [http://localhost:3000/login](http://localhost:3000/login)
4. Introduce the mail test2@test.test
5. Introduce the password Test2@test
6. Click Acceder
7. Go to Mi Perfil
8. Go to Mis Anuncios

Expected result:
* Show list of ads.

Actual result:
* Error message: “Ha habido un error con tu petición”.

`Status Code: 500 Internal Server Error`
